docs/api.rst

```diff
@@ -4991,6 +4975,106 @@ Dealing with History Events
    :statuscode 409: No user is logged in or one of the identifiers to delete did not correspond to an event in the DB or one of the identifiers was for the last event in the corresponding transaction hash and force_delete was false..
    :statuscode 500: Internal rotki error
 
+.. _filter-request-args-label:
```

Comment:
This shouldn't be in the middle of all endpoints. It should be at the very end. Also did you try to do `make html` and look at the generated docs to make sure the link works fine and renders fine?